### PR TITLE
web: Display current working directory on status-page

### PR DESCRIPTION
**Pull request by msiebuhr**
Show current working directory in /status (ref #2247).

Ping @beorn7 @fabxc @juliusv

### CODE REVIEW

**Comment by brancz** [comment]
lgtm

File: web/web.go [hunk 1]
```diff
@@ -127,6 +128,8 @@ func New(o *Options) *Handler {
 		return o.Context, nil
 	})
 
+	cwd, _ := os.Getwd()
```
Comment (fabxc) [inline web/web.go]: We shouldn't drop the error here. We can set `cwd = "<error retrieving working directory>"` and log the error.

**Comment by fabxc** [comment]
I suppose with the current level of information we provide, the working directory add any security concerns. But at least one other person should confirm that.

**Comment by beorn7** [comment]
I have no concern with displaying the word directory on the status page.

**Comment by fabxc** [comment]
Unfortunately this needs a rebase as the generated bindata was modified in master by another change.

**Comment by msiebuhr** [comment]
@fabxc re-based, re-generated and re-pushed...

**Comment by fabxc** [comment]
Could you address my comment on the error handling?

**Comment by msiebuhr** [comment]
@fabxc Sorry about that - had a lot going on today :)

**Comment by fabxc** [comment]
Great! Thank you 👍

**Pull request merged**